op-node/rollup/status/status.go

```diff
@@ -66,6 +66,7 @@ func (st *StatusTracker) OnEvent(ev event.Event) bool {
 		st.log.Debug("Forkchoice update", "unsafe", x.UnsafeL2Head, "safe", x.SafeL2Head, "finalized", x.FinalizedL2Head)
 		st.data.UnsafeL2 = x.UnsafeL2Head
 		st.data.SafeL2 = x.SafeL2Head
+		st.data.LocalSafeL2 = x.SafeL2Head
```

Comment:
Not sure about this one. We may inadvertently reset the local safe to an earlier value. Local safe can be ahead, and an FCU calls' _safe_ only refers to cross-safe.

Reply:
How about like this? [7593391](https://github.com/ethereum-optimism/optimism/pull/16296/commits/759339124fb5aef25f5e41a40d8ca4d906964538)